### PR TITLE
feat: added tags to script model

### LINKED ISSUE #623: Tags - backend - add tags support to Script
Apply tags in: 
- [x] open api spec + generate model
- [x] filters in API 
- [x] use filters in API `/executions?tags=blabl,nlabal,blabl`
- [x] use tags filters in API `/scripts`
- [ ] expose `/tags` for dashboard 
- [x] change direct client to allow tags passing 
- [x] change proxy client to allow tags passing
- [ ] #676
- [x] kubectl-testkube plugin to allow to add tags on script create / update.

### PR DESCRIPTION
**Pull request by jasmingacic**
Signed-off-by: Jasmin Gacic <jasmin.gacic@gmail.com>

## Pull request description 

Just the model and API definition

Fixes #623 

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: api/v1/testkube.yaml [hunk 1]
```diff
@@ -41,6 +40,11 @@ paths:
         - $ref: "#/components/parameters/ExecutionsStatusFilter"
         - $ref: "#/components/parameters/ExecutionsStartDateFilter"
         - $ref: "#/components/parameters/ExecutionsEndDateFilter"
+        - in: query
```
Comment (exu) [inline api/v1/testkube.yaml:43]: make it reference

File: api/v1/testkube.yaml [hunk 2]
```diff
@@ -225,6 +227,12 @@ paths:
       tags:
         - scripts
         - api
+      parameters:
```
Comment (exu) [inline api/v1/testkube.yaml:230]: use ref

**Review by exu — CHANGES_REQUESTED** [review]

File: api/v1/testkube.yaml [hunk 3]
```diff
@@ -936,6 +941,11 @@ components:
         created:
           type: string
           format: date-time
+        tags:
+          type: array
+          items:
+            type: string
+          description: test tags
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply api/v1/testkube.yaml:948]:
```suggestion
          description: script tags
```

File: api/v1/testkube.yaml [hunk 4]
```diff
@@ -831,18 +834,23 @@ components:
...
           type: array
           description: "steps execution restults"
           items:
             $ref: "#/components/schemas/TestStepExecutionResult"
             description: test execution results
+        tags:
+          type: array
+          items:
+            type: string
+          description: test tags
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply api/v1/testkube.yaml:853]:
```suggestion
          description: test execution tags
```

File: api/v1/testkube.yaml [hunk 5]
```diff
@@ -983,16 +993,21 @@ components:
...
           format: date-time
         executionResult:
           required: true
           description: result get from executor
           $ref: "#/components/schemas/ExecutionResult"
+        tags:
+          type: array
+          items:
+            type: string
+          description: test tags
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply api/v1/testkube.yaml:1010]:
```suggestion
          description: execution tags
```

**Review by exu — APPROVED** [review]

**Review by fog1985 — APPROVED** [review]
LGTM

**Pull request merged**